dashboard/app/mailers/pd/application/teacher_application_mailer.rb

```diff
@@ -1,6 +1,6 @@
 module Pd::Application
   class TeacherApplicationMailer < ActionMailer::Base
-    CODE_ORG_DEFAULT_NOTIFICATION_EMAIL = 'Jared Fritz <jared@code.org>'
+    CODE_ORG_DEFAULT_NOTIFICATION_EMAIL = 'Becky Kenemuth <teacher@code.org>'
```

Comment:
Are we not setting a personal email as the return anymore?

Reply:
Yeah we are just making it teacher@code.org now!